test/unit/test_distributed.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License'). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the 'license' file accompanying this file. This file is
+# distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+from mock import call, patch
+
+from sagemaker_mxnet_container.distributed import DefaultParameterServer
+
+SCHEDULER = 'host-1'
+SINGLE_HOST_LIST = [SCHEDULER]
+MULTIPLE_HOSTS_LIST = [SCHEDULER, 'host-2', 'host-3']
+
+IP_ADDRESS = '0.0.0.0000'
+DEFAULT_PORT = '8000'
+DEFAULT_VERBOSITY = '0'
+BASE_ENV_VARS = {
+    'DMLC_NUM_WORKER': str(len(MULTIPLE_HOSTS_LIST)),
+    'DMLC_NUM_SERVER': str(len(MULTIPLE_HOSTS_LIST)),
+    'DMLC_PS_ROOT_URI': IP_ADDRESS,
+    'DMLC_PS_ROOT_PORT': DEFAULT_PORT,
+    'PS_VERBOSE': DEFAULT_VERBOSITY,
+}
+
+MXNET_COMMAND = "python -c 'import mxnet'"
+
+
+def test_init():
```

Comment:
i don't see analogous test for multiple hosts case

Reply:
added